app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -601,3 +624,30 @@ fun CreateActivityScreen(
             selectedItem = Route.ADD_ACTIVITY)
       })
 }
+
+@Composable
+fun RemainingPlace(field: String, maxLength: Int) {
+  Row(
+      modifier =
+          Modifier.fillMaxWidth().padding(horizontal = MEDIUM_PADDING.dp).testTag("remainingPlace"),
+      horizontalArrangement = Arrangement.End) {
+        Column(
+            horizontalAlignment = Alignment.End,
+            modifier = Modifier.testTag("remainingPlaceColumn")) {
+              Text(
+                  text = "${field.length}/$maxLength characters",
+                  fontSize = MEDIUM_PADDING.sp,
+                  color = Color.Gray,
+                  modifier = Modifier.testTag("remainingPlaceText"))
+              LinearProgressIndicator(
+                  progress = field.length / maxLength.toFloat(),
+                  modifier =
+                      Modifier.height(STANDARD_PADDING.dp)
+                          .width(130.dp)
+                          .clip(RoundedCornerShape(SMALL_PADDING.dp))
+                          .testTag("remainingPlaceProgress"),
+                  color = Color(DARK_BLUE_COLOR),
+                  backgroundColor = Color.LightGray)
+            }
+      }
```

Comment:
Good UI improvement that will make it easier for the user to see which activities are the most close to be filled